### DISCUSSION TITLE
Jenkins Provision job's error - Provision/Core/PostgresDbUpdate

### DISCUSSION THREAD

**Discussion by bsimmonayyakutti** [project-sunbird/sunbird-devops#3499] [General]
TASK [postgresql-data-update : Install postgres client to create schema from file] ***
fatal: [10.128.0.23]: FAILED! => {
    "changed": false
}

MSG:

Failed to update apt cache: W:Updating from such a repository can't be done securely, and is therefore disabled by default., W:See apt-secure(8) manpage for repository creation and user configuration details., W:Target Packages (multiverse/binary-amd64/Packages) is configured multiple times in /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target Packages (multiverse/binary-all/Packages) is configured multiple times in /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target Translations (multiverse/i18n/Translation-en) is configured multiple times in /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target CNF (multiverse/cnf/Commands-amd64) is configured multiple times in /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, W:Target CNF (multiverse/cnf/Commands-all) is configured multiple times in /etc/apt/sources.list.d/mongodb-org-4.4.list:1 and /etc/apt/sources.list.d/repo_mongodb_org_apt_ubuntu.list:1, E:The repository 'http://www.apache.org/dist/cassandra/debian 39x Release' does not have a Release file.

PLAY RECAP *********************************************************************
10.128.0.23                : ok=6    changed=4    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

**Comment by dikshauat** [comment]
Hi, I am also getting the same error. Please let me know if you find any resolve for it. 

Regards
